source/configuration_manual/mail_location/obox/amazon_s3.rst

```diff
@@ -60,10 +76,15 @@ by adding the bucket region parameter to the S3 URL:
 .. code-block:: none
 
   plugin {
-    obox_index_fs = compress:gz:6:s3: https://ACCESSKEY:SECRET@host/?bucket=BUCKETNAME&region=eu-central-1
+    obox_index_fs = compress:gz:6:aws-s3: https://ACCESSKEY:SECRET@host/?bucket=BUCKETNAME&region=eu-central-1
   }
 
-The s3 driver also supports the bulk-deletion. The bulk-delete is enabled by default to delete up to 1000 keys with one request. To change this behaviour refer to ``bulk_delete_limit`` at :ref:`http_based_object_storages`
+The ``aws-s3`` and ``s3`` drivers support bulk-deletion. The ``bulk-delete``
+option is enabled by default to delete up to 1000 keys with one request.
+To change this behaviour refer to ``bulk_delete_limit`` at
+:ref:`http_based_object_storages`. Bulk delete can only efficiently run on
+multiple objects if configured to do so, via setting
+``obox_max_parallel_deletes`` greater one (refer to :ref:`obox_settings`).
 
   .. versionadded:: 2.3.10
```

Comment:
It should also say how s3 and aws-s3 differ (list the default parameters that it uses for s3).